### PR TITLE
Move Math-Complex from cpan/ to dist/

### PR DESCRIPTION
**Pull request by khwilliamson**
This module is now being maintained by p5p now.

### CODE REVIEW

**Comment by leonerd** [comment]
This seems to be fine... Are we waiting on any specific comments or reviews? How does PAUSE permissions for this alter - do we need to talk to the CPAN admins about it?

**Comment by Grinnz** [comment]
P5P has first-come on the modules, so permissions can be given to a releaser as needed. Someone(tm) should release the distribution to CPAN once it has been moved since dist/ entries are not indexed in perl releases.

File: t/porting/customized.dat
Comment (bram-perl) [inline t/porting/customized.dat]: You may need to double check the list of removed customizations, it removes some that are unrelated to Math::Complex
Also, in #20216 @tonycoz noted that customization of 'Filter::Util::Call' is still present (**I** did not verify)

**Comment by Leont** [comment]
Are we sure we want to do this? IMO we should generally prefer to have modules on CPAN unless there's a reason to do otherwise (e.g. when it's tied to core development the way Devel::PPPort is). Delegating maintenance of other modules is the more manageable option IME.

**Comment by jkeenan** [comment]
> Are we sure we want to do this? IMO we should generally prefer to have modules on CPAN unless there's a reason to do otherwise (e.g. when it's tied to core development the way Devel::PPPort is). Delegating maintenance of other modules is the more manageable option IME.

If @pjacklam is willing to be the maintainer of this module on CPAN, then I would endorse that approach.  He would be able to apply patches and test them out against older Perl versions directly, without having to wait around for us (P5P) to approve his patches.

Only complication is that we haven't heard from the module's current CPAN maintainer in years.  That's why I think the Perl Steering Council should discuss this with PAUSE administrators.

**Comment by khwilliamson** [comment]
On 9/2/22 14:48, James E Keenan wrote:
>     Are we sure we want to do this? IMO we should generally prefer to
>     have modules on CPAN unless there's a reason to do otherwise (e.g.
>     when it's tied to core development the way Devel::PPPort is).
>     Delegating maintenance of other modules is the more manageable
>     option IME.
> 
> If @pjacklam <https://github.com/pjacklam> is willing to be the 
> maintainer of this module on CPAN, then I would endorse that approach. 
> He would be able to apply patches and test them out against older Perl 
> versions directly, without having to wait around for us (P5P) to approve 
> his patches.
> 
> Only complication is that we haven't heard from the module's current 
> CPAN maintainer in years. That's why I think the Perl Steering Council 
> should discuss this with PAUSE administrators.
> 
>
I thought @pjacklam said that the PAUSE administrators were not 
responding or not responding favorably to his wanting to be comaint.,

If I am right, that is unconscionable, and the system needs immediate reform

**Comment by pjacklam** [comment]
The PAUSE admins replied that I should contact the maintainers again. However, as I had contacted the maintainers in 2015 and again in 2020 without success, I didn't see any point in contacting them yet again. So I gave up Math-Complex and created my own modules for elementary functions, spherical trigonometry etc. (I have a background in applied mathematics and numerical analysis and work with air navigation).

The reason why I have submitted pull requests recently is that I noticed Math-Complex in core (1.59_02)  is ahead of the version on CPAN (1.59). I thought my pull requests might be a way to get some progress on Math-Complex.

It is not important for me to be the one in charge of Math-Complex. I just wish there was someone, not necessarily me. If I remember correctly, @shlomif also wanted co-maint some years ago. Perhaps he is still interested?

**Comment by khwilliamson** [comment]
@leonerd @rjbs @book 
We need some action on this

**Comment by rjbs** [comment]
@khwilliamson You should just move this to dist, thanks.  If we get a strong contender for fixing things, we can move it back (easy) or just accept a bunch of fixes to the library.

[ edit:  I originally wrote "to cpan", which was just a brain-to-fingers error ]

**Review by rjbs — APPROVED** [review]

**Comment by khwilliamson** [comment]
On 9/30/22 08:26, Ricardo Signes wrote:
> @khwilliamson <https://github.com/khwilliamson> You should just move 
> this to cpan, thanks. If we get a strong contender for fixing things, we 
> can move it back (easy) or just accept a bunch of fixes to the library.

It already is in cpan.  I presume you meant move it to dist

File: t/porting/customized.dat
Comment (khwilliamson) [reply t/porting/customized.dat]: fixed

**Pull request merged**
